app/templates/resources/read_resource.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td><a target="_blank" href="{{ resource.website }}">{{ resource.website }}</a></td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.full_name() }}</td></tr>
+            </table>
+            <a href="{{ url_for('resources.create_review', resource_id=resource.id) }}">
+                <button class="ui button">Write a Review</button>
+            </a>
+        </div>
+
+        {% for review in resource.reviews.all() %}
+        <div class="twelve wide column">
+            <h1>Reviews</h1>
+            <h3>User ID: {{ review.user_id }}</h3>
+            <table class="ui compact definition table" id="{{ review.id }}">
+                <tr><td>Content</td>
+                    <td>{{ review.content }}</td></tr>
+                <tr><td>Rating</td>
+                    <td>{{ review.rating }}</td></tr>
+            </table>
+            {% if review.user_id is equalto current_user_id %}
```

Comment:
if we are going to pass in the current_user_id, then let's just pass in a variable that is `is_current` and you can set is_current in the views.py by saying `is_current = review.user_id == current_user.id`